tests/backup/backup_restore_basic_test.go

```diff
@@ -936,24 +937,31 @@ var _ = Describe("{CustomResourceRestore}", func() {
 
 var _ = Describe("{AllNSBackupWithIncludeNewNSOption}", func() {
 	var (
-		contexts                   []*scheduler.Context
-		cloudCredUID               string
-		cloudCredName              string
-		backupLocationName         string
-		backupLocationUID          string
-		backupLocationMap          map[string]string
-		periodicSchedulePolicyName string
-		periodicSchedulePolicyUid  string
-		scheduleName               string
-		appNamespaces              []string
-		scheduleNames              []string
-		appClusterName             string
-		restoreName                string
-		nextScheduleBackupName     interface{}
+		contexts               []*scheduler.Context
+		newContexts            []*scheduler.Context
+		cloudCredUID           string
+		cloudCredName          string
+		backupLocationName     string
+		backupLocationUID      string
+		backupLocationMap      map[string]string
+		schedulePolicyName     string
+		schedulePolicyUid      string
+		scheduleName           string
+		appNamespaces          []string
+		newAppNamespaces       []string
+		scheduleNames          []string
+		restoreName            string
+		nextScheduleBackupName interface{}
+		intervalInMins         int
+		numDeployments         int
 	)
 
 	JustBeforeEach(func() {
 		StartTorpedoTest("AllNSBackupWithIncludeNewNSOption", "Verification of schedule backups created with include new namespaces option", nil, 84760)
+		numDeployments = Inst().GlobalScaleFactor
+		if len(Inst().AppList) == 1 && numDeployments < 2 {
+			numDeployments = 2
+		}
```

Comment:
We need not do this

Reply:
Even if there is only one app in the app list, I want this test to create two namespaces before creating a schedule backup.